.github/workflows/flowzone.yml

```diff
@@ -11,12 +11,7 @@ jobs:
   flowzone:
     name: Flowzone
     uses: product-os/flowzone/.github/workflows/flowzone.yml@master
-    secrets:
-      FLOWZONE_TOKEN: ${{ secrets.FLOWZONE_TOKEN }}
-      GPG_PRIVATE_KEY: ${{ secrets.GPG_PRIVATE_KEY }}
-      GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
-      BALENA_API_KEY: ${{ secrets.BALENA_TOKEN }}
```

Comment:
This one is problematic as the name doesn't match up properly, `BALENA_TOKEN` vs `BALENA_API_KEY`

Reply:
Checks are still failing because BALENA_TOKEN is blank from the deploy-to-balena action. I guess the 2 solutions forward to pushing this through are:

- Flowzone should pass both BALENA_API_KEY as BALENA_TOKEN to the GitHub action to conform to its interface
- The GitHub action should accept both BALENA_API_KEY and BALENA_TOKEN

Reply:
PR: https://github.com/product-os/flowzone/pull/256

Gonna push a test commit to see if this works (I except it will)

Reply:
@cywang117 I think the correct answer is to add `BALENA_API_KEY` to the balena-os secrets, the problem is I don't know where the contents of `BALENA_TOKEN` for balena-os came from in order to add the extra API key - if either you or @pipex know where they came from/who added them then I can handle the rename

Reply:
Converting to draft real quick so this doesn't get auto-merged

Reply:
I'm not quite sure where, the secret isn't in the repository level I'm guessing @Page- ?

Reply:
It's there at the org level @cywang117 but I cannot rename it or access the contents so I need to get the contents anew from the original source in order to add it as a "new" secret called `BALENA_API_KEY`